python/torch_mlir_e2e_test/test_suite/basic.py

```diff
@@ -64,6 +64,48 @@ def BmmModule_basic(module, tu: TestUtils):
 # ==============================================================================
 
 
+class is_floating_point_1(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1], torch.int32, True),
```

Comment:
`tu.rand` generates a float tensor, so this should be `float`. If you want an int tensor, use `torch.randint`